package.json

```diff
@@ -9,6 +9,7 @@
         "test": "jest"
     },
     "dependencies": {
+        "leaflet.heightgraph": "github:easbar/Leaflet.Heightgraph#61cd2e91935719caa93a9c73a6940c247e3212a0",
```

Comment:
This should be ok for now, but in the longer run we should either clean up this repository so it can be used standalone or move the code we need to the maps repository.